lib/muchos/config/base.py

```diff
@@ -216,21 +260,9 @@ def checksum_ver(self, software, version):
             exit('ERROR - Failed to find checksums for %s %s in %s' % (software, version, self.checksums_path))
         return self.checksums_d[key]
 
-    def verify_instance_type(self, instance_type):
-        if not self.cluster_template_d:
-            if get_arch(instance_type) == 'pvm':
-                exit("ERROR - Configuration contains instance type '{0}' that uses pvm architecture."
-                     "Only hvm architecture is supported!".format(instance_type))
-
+    @abstractmethod
     def instance_tags(self):
```

Comment:
I think that would be fine since azure doesn't have this defined yet.  I kept it as part of the base interface because we might want to make it available for use with azure but on second thought, 'existing' wouldn't have tags at all so it really doesn't make sense to require it for all cluster types.  So, yes, I will move it to ec2 config